docs/pricing/free.mdx

```diff
@@ -0,0 +1,40 @@
+| 200 chat messages and prompts per month                                       | All popular coding languages                        | -                  | -                                          | -                      |
+| Multiple LLM selection (Claude 3.5 Sonnet, Gemini 1.5 Pro and Flash, Mixtral) | Natural language search                             | -                  | -                                          | -                      |
+| Support for local Ollama models                                               | All major codehosts (GitHub, GitLab, Bitbucket)     | -                  | -                                          | -                      |
+
+## Pricing and billing cycle
+
+There is no billing cycle, as it's free to use and supports one user per account. If you exceed your daily limits, you will have to wait until the end of the month to use the feature again. You can upgrade to our Enterprise Starter plan for more advanced features and usage limits.
+
+## Free vs. Enterprise Starter comparison
+
+The Enterprise Starter plan provides extended usage limits and advanced features compared to the Free plan. Here's a side-by-side comparison of the two:
+
+|       **Feature**        |                                                                         **Free**                                                                         |                                                                  **Enterprise Starter**                                                                   |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Description**              | - AI editor assistant for hobbyists or light usage                                                                                                       | - AI and search for growing organizations hosted on our cloud                                                                                             |
+| **Price**                    | - $0/month <br /> - 1 user                                                                                                                               | - $19/user/month <br /> - Up to 50 devs                                                                                                                   |
+|**AI features**              | - Unlimited autocomplete <br /> - 200 chat messages and prompts per month <br /> - Multiple LLM choices for chat <br /> - Connect to local Ollama models | - Unlimited autocomplete and chat <br /> - More powerful LLMs for chat (GPT-4o, Claude 3 Opus) <br /> - Intent detection and integrated search results    |
```

Comment:
ES will not have unlimited auto-complete or chat. I think we are starting with 500 autocompletes a day and 75 chats a day but I am not sure we should list this here since we will likely adjust. Could we drop this or say something  softer?  /cc @eseliger

Reply:
Let's mention the number of these limits rather than being open-ended. @eseliger, can you please confirm these numbers?

Reply:
We got feedback before to not be explicit here. So unless you feel strongly (and then I would start a slack thread to discuss) let omit. I also dont think we explicitly document enterprise limits.

Reply:
Gotcha let's omit then.